iocage/lib/Jail.py

```diff
@@ -372,6 +372,89 @@ def start(
 
         self._run_hook("poststart")
 
+    def fork_exec(
+        self,
+        command: typing.List[str],
+        error_handler: typing.Optional[typing.Callable[
+            [typing.Tuple[subprocess.Popen, str, str]],
+            typing.Tuple[bool, str],
+        ]]=None,
+        **temporary_config_override
+    ) -> typing.Generator['iocage.lib.events.IocageEvent', None, None]:
+        """
+        Starts a jail, runs a command and exists
```

Comment:
perhaps explain why the jail needs to be stopped before using this function…
also: does the function stop the jail at the end?